src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -101,7 +103,7 @@ public Receiver2()
                 EndpointSetup<DefaultServer>(c =>
                 {
                     c.EndpointName("DistributingACommand.Receiver");
-                    c.ScaleOut().InstanceDiscriminator("2");
+                    c.EndpointInstanceId(_ => "2");
                 });
```

Comment:
The idea is that this user-facing API should guide users against hard-coding that value. That's why it get's a func.